tests/scripts/test_scheduler_with_profiling.sh

```diff
@@ -39,7 +48,8 @@ function run_without_sched()
 
     RESULT=$(get_result_of_benchmark_test $BENCHMARK_DRIVER_BIN $MODEL $LOG_FILE)
 
-    printf -v RESULT_INT '%d' $RESULT 2>/dev/null
```

Comment:
It returns error code because `RESULT` is float.

```
$ printf -v RESULT %d 20.1
bash: printf: 20.1: invalid number
```

Reply:
I think you may use `.0f`.

```
$ printf -v RESULT %.0f 20.1
$ echo $?
0
$ echo $RESULT
20
```
But it rounds to near-to-zero while '%d' truncate numbers after decimal-point. (= `floor` for positive number ).

Reply:
Good. I'll update this.